h/api/search/test/config_test.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+import itertools
+import re
+import urllib
+
+from h.api.search.config import ANNOTATION_ANALYSIS
+
+
+def test_strip_scheme_char_filter():
```

Comment:
It's mainly to make a search for "example.com/foo/bar" match annotations where the URI field is "https://example.com/foo/bar," I expect.